### PR TITLE
allow Symfony 4.0 requirements

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by GuilhemN** [comment]
Shouldn't we run tests on it?

**Comment by xabbuh** [comment]
done

File: composer.json [hunk 1]
```diff
@@ -30,40 +30,41 @@
...
+        "symfony/twig-bundle": "^2.7|^3.0|^4.0",
+        "symfony/browser-kit": "^2.7|^3.0|^4.0",
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
+        "symfony/expression-language": "~2.7|^3.0|^4.0",
+        "symfony/css-selector": "^2.7|^3.0|^4.0",
         "phpoption/phpoption": "^1.1",
         "jms/serializer-bundle": "^1.0",
         "psr/http-message": "^1.0"
     },
+    "minimum-stability": "dev",
```
Comment (GuilhemN) [inline composer.json]: Can't we just require `4.0.*@dev` instead ?

Comment (xabbuh) [inline composer.json]: We can if you prefer that. :)

Comment (GuilhemN) [inline composer.json]: It's fine for me as long as we don't have problems, just tests are more likely to break with this setting.
Let's keep it this way for now, we'll see soon enough if it causes issues :)

**Comment by GuilhemN** [comment]
Thank you @xabbuh!

File: composer.json (hunk 1 above)
Comment (xabbuh) [reply composer.json]: see #1800

**Pull request merged**